### ISSUE TITLE
[BUG] Docbloack changed (wrongly) without a real reason

### ISSUE THREAD

**Issue by Aerendir** [rectorphp/rector#2942]
# Bug Report

| Subject        | Details                                                         |
| :------------- | :---------------------------------------------------------------|
| Rector version | v0.7.1              |
| Installed as   | composer dependency                      |

Rector changes the Docblocks without doing nothing.

This causes issues with PHPCsFixer and with Phan.

PHPCSFixer errors:

>   2) src/Repository/UserRepository.php (no_spaces_inside_parenthesis, unary_operator_spaces, not_operator_with_space, phpdoc_align)

Phan errors:

```console
src/Repository/UserRepository.php:29 PhanUnextractableAnnotation Saw unextractable annotation for comment '* @method (User | null) find($id, $lockMode = null, $lockVersion = null)'
src/Repository/UserRepository.php:30 PhanUnextractableAnnotation Saw unextractable annotation for comment '* @method (User | null) findOneBy(array $criteria, array $orderBy = null)'
```

## Minimal PHP Code Causing Issue

https://getrector.org/demo/5548eb18-1fce-4208-937c-076e3c5ca986

```diff
 /**
- * @method User|null find($id, $lockMode = null, $lockVersion = null)
- * @method User|null findOneBy(array $criteria, array $orderBy = null)
+ * @method (User | null) find($id, $lockMode = null, $lockVersion = null)
+ * @method (User | null) findOneBy(array $criteria, array $orderBy = null)
  * @method User[]    findAll()
  * @method User[]    findBy(array $criteria, array $orderBy = null, $limit = null, $offset = null)
- */
+ */
 class UserRepository
 {
 }
```

## Expected Behaviour

Don't change the dockblock comment.

When it has to be changed, use standard annotations and not this one with parenthesis that causes troubles with other softwares.

**Comment by TomasVotruba** [comment]
Could you try `dev-master`?

The method one was fixed here: https://github.com/rectorphp/rector/commit/6fce4dcd37189fc6e9c4855f7a74b09e802cde6f

**Comment by TomasVotruba** [comment]
Released as v0.7.2 just now: https://github.com/rectorphp/rector/releases/tag/v0.7.2
